### PR TITLE
Move checkbox labels to the left of the checkboxes in create VM dialog

### LINKED ISSUE #45: machines: Create new VM dialog is missing a label from the options checkboxes
The create VM dialog implies that the install and run options are part of the memory settings.

Screenshot from 2021-03-15 11-50-21 memory options

The group of options should have their own label.

This is both a UX and accessibility issue, thankfully with a simple fix.

### PR DESCRIPTION
**Pull request by KKoukiou**
Fixes #45

### CODE REVIEW

**Review by marusak — APPROVED** [review]
Thanks!

File: src/scripts/install_machine.py [hunk 1]
```diff
@@ -129,16 +129,16 @@ def prepare_cloud_init(args):
             mode='w+'
         )
         user_data_file.write("#cloud-config\n")
-        if 'userLogin' in args:
+        if args['userLogin']:
```
Comment (marusak) [inline src/scripts/install_machine.py:132]: This is already merged in https://github.com/cockpit-project/cockpit-machines/commit/b3681c910006332e1cea2805cbbfc0b7986ae50f

Yet this commit also changes references. Github does not seem to see that as conflict. This is weird. I would say it needs rebase (and the changed refs should be in different commit?)

**Comment by KKoukiou** [comment]
This has failing pixel tests - can take a look after my PTO.

**Comment by marusak** [comment]
Fixed the reference images

**Review by marusak — APPROVED** [review]
Thank you!
